nss-day-cohort-39/tabloid-full-stack-velours
# profile image upload

### Client-Side Changes
- Added upload image button, selected file field, and save link to UserProfileDetails.js, that only shows up if you are looking at your own user details
- Added return to getToken() in editUserProfile() in UserProfileProvider.js to complete the fetch before doing executing .then in UserProfileDetails.js
- Styled User Profile Info page
- Added refreshUser() for automatic page re-render one image is saved

### Testing
- Login as an admin and click on "Users" in Header
- Click on the i info icon next to your own name and make sure upload image button appears toward bottom
- Change your profile image, using a .jpg or .png file
- Make sure file name appears to the right of the Upload Image button
- Click save, make sure page re-renders with your new profile image
- Click on a different user and make sure upload button does not appear
